public/javascripts/nodemailer.js

```diff
@@ -1,15 +0,0 @@
-require("dotenv").config();
-const nodemailer = require('nodemailer');
-
-const nodemailerSetup = async () => {
-  let transporter = await nodemailer.createTransport({
-    service: "Gmail", // can switch for 'hotmail'
-    auth: {
-      user: "chrisjcastle93@gmail.com",
```

Comment:
I wouldn't publish your personal email address like this just out of security, best to stick it in a `env` var